backend/test/card_game/effects_test.clj

```diff
@@ -1,33 +1,48 @@
 (ns card-game.effects-test
   (:require [expectations.clojure.test :refer :all]
-            [card-game.core :as core]))
+            [card-game.core :as core]
+            [configs :as configs]))
 
 (defexpect card-altering
   ; Altering a card in hand works
   (expect
-    [15 10 10 10 10 10 10 10 10 10 10 15]
+    (concat [9001] (subvec (vec (map :power (configs/ini-hand))) 2))
     (map :power
          (-> (core/new-game)
-             (core/alter-card [:players 0 :hand 0] {:power 15})
-             (core/alter-card [:players 0 :hand 11] {:power 15})
+             (core/alter-card [:players 1 :hand 0] {:power 9001})
+             (core/play-card 0 0 0)
+             (core/play-card 1 1 1)
+             :players
+             (nth 1)
+             :hand)))
+
+  (expect
```

Comment:
I think a few `(let` would make this test much more readable. (for example setting `last-card (dec (count (configs/ini-hand)))`

Reply:
any suggestion will be welcomed